website/docs/Use-Cases/agent_chat.md

```diff
@@ -39,16 +39,16 @@ assistant = AssistantAgent(name="assistant")
 # create a UserProxyAgent instance named "user_proxy"
 user_proxy = UserProxyAgent(name="user_proxy")
 ```
-#### Function calling
+#### Tool calling
```

Comment:
I believe this is still called function calling. Currently, there are four types of tools defined in [Assistants tools](https://platform.openai.com/docs/assistants/tools):
- [Code Interpreter](https://platform.openai.com/docs/assistants/tools/code-interpreter)
- [Knowledge Retrieval](https://platform.openai.com/docs/assistants/tools/knowledge-retrieval)
- [Function Calling](https://platform.openai.com/docs/assistants/tools/function-calling)
- [Supported Files](https://platform.openai.com/docs/assistants/tools/supported-files)

The competition API supports only function calling, but for consistency it is probably as the only tool available.

Reply:
Yeah I don't know which is clearer to call out for headers and documentation, happy to change it back if no other comments

Reply:
I think "function calling" is a better term here, but it is presumably subjective and would like to hear other opinions.

Reply:
I think it is better to wrap up this PR as it is blocking some other PRs.